File: app/src/main/java/ua/syt0r/furiganit/ui/settings/SettingsFragment.java
package ua.syt0r.furiganit.ui.settings;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;

import ua.syt0r.furiganit.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
    }
}

File: app/src/main/java/ua/syt0r/furiganit/app/main/screens/settings/SettingsFragment.kt
package ua.syt0r.furiganit.app.main.screens.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import org.koin.androidx.viewmodel.ext.android.viewModel
import ua.syt0r.furiganit.R

class SettingsFragment : PreferenceFragmentCompat() {

	private val settingsViewModel: SettingsViewModel by viewModel()

	override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
		setPreferencesFromResource(R.xml.preferences, rootKey)
	}
}
